doc/dftb+/manual/dftbp.tex

```diff
@@ -3911,133 +3914,163 @@ \subsection{SSR22}
 This tag contains the specifications for a DFTB/SSR(2,2) calculation~\cite{Lee_JCTC_2019}, based on ensemble DFT theory.
 
 \textbf{Note:} the \dftbp{} binary can be compiled with OpenMP (not MPI) parallelization
-and ensemble DFT calculation is not compatible with time-dependent DFTB calculation.
-DFTB/SSR is not compatible with spin-polarisation, but it requires spin constants.
+and DFTB/SSR calculation is not compatible with time-dependent DFTB calculation.
+In addition, it is not compatible with spin-polarisation, but it requires spin constants to treat open-shell microstates.
 
 In general, REKS calculation can be classified as single-state REKS, SA-REKS and SI-SA-REKS.
 In single-state REKS, only ground state is calculated and it can treat the state with multireference
 character. SA-REKS and SI-SA-REKS can calculate the vertical excitation energies. The difference
 is that the state-interaction term is considered in SI-SA-REKS so that more accurate states can
 be generated. The corresponding oscillator strengths as well as excited state geometry optimisation
 can be performed with these options, details of the resulting output files are given in appendix
-\ref{sec:reks_files}. DFTB/SSR is currently implemented for the LC-DFTB level of theory with external
-point charges or dispersion corrections. Third order SCC calculation can be considered in only
-energy evaluation. The gamma point periodic systems can be calculated using DFTB/SSR without
-range-separated functionals. Finally, only single-state REKS can calculate the stress as well as
-lattice optimization. The specifications for this block have the following properties:
+\ref{sec:reks_files}.
+
+In the context of DFTB, current REKS calculation is compatible with following functionalities.
+The range-separated functional, external point charges and dispersion corrections can be
+calculated with single-state REKS, SA-REKS or SI-SA-REKS. For the periodic system, only
+gamma point sampling is supported with REKS. Especially, the stress evaluation and lattice
+optimisation is possible with only single-state REKS.
+The specifications for this block have the following properties:
+
+\begin{ptable}
+  \kw{Energy} & m & & \cb & \\
+  \kw{TargetState} & i & & 1 & \\
+  \kw{TargetMicrostate} & i & Functional $\neq$ \{ "PPS" \}, StateInteractions = No & 0 & \\
+  \kw{ReadEigenvectors} & l & & No \\
+  \kw{FonMaxIter} & i & & 20 & \\
+  \kw{Shift} & r & & 0.3 & \\
+  \kw{SpinTuning} & $(\text{r})^*$ & & \cb & \\
+  \kw{TransitionDipole} & l & Functional $\neq$ \{ "PPS" \}, TargetMicrostate = 0 & No & \\
+  \kw{Gradient} & m & & ConjugateGradient \cb & \\
+  \kw{RelaxedDensity} & l & & No & \\
+  \kw{NonAdiabaticCoupling} & l & Functional $\neq$ \{ "PPS" \}, StateInteractions = Yes & No & \\
+  \kw{VerbosityLevel} & i & & 1 & \\
+\end{ptable}
```

Comment:
In the pdf file, this part shows too long length for the width. Is there a good idea for this table?

Reply:
A parbox might solve this for the moment, something like
\parbox{0.3\textwidth}{Functional $\neq$ \{ "PPS" \}, TargetMicrostate = 0}

In the longer term, we probably should re-define the table macro to set that column as a p{width}

Reply:
Thanks. This problem is solved by using \parbox. Now, it shows normal length for the width in REKS keyword tables.